### PR TITLE
[author] add geolocator@2.0.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by onury**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/onury/geolocator
 * Official website (optional, not the repository): https://onury.github.io/geolocator/
 * NPM package url (optional): https://www.npmjs.com/package/geolocator
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 228
   - Count of watchers: 14
   - Count of forks: 52
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 108
   - Downloads in the last month: 400

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@onury Thanks, however I found that this PR touched something unrelated like license info in package.json, what about just leave the unrelated part alone for a clean and clear commit/change history? Thanks.

**Comment by onury** [comment]
@PeterDaveHello updated.

**Comment by PeterDaveHello** [comment]
Thanks @onury

File: ajax/libs/geolocator/package.json [hunk 1]
```diff
@@ -1,33 +1,49 @@
...
+    "lng",
+    "lon"
   ],
   "license": "MIT",
-  "author": "Onur Yıldırım",
+  "author": {
+    "name": "Onur Yıldırım",
+    "email": "onur@cutepilot.com"
+  },
+  "homepage": "https://github.com/onury/geolocator",
```
Comment (PeterDaveHello) [inline ajax/libs/geolocator/package.json]: Oops, this is no need actually.

**Comment by onury** [comment]
@PeterDaveHello deleted homepage. thanks.

**Comment by PeterDaveHello** [comment]
No mind, merged

**Pull request merged**